Recurrent time out failure in Travis

We have recurrent failures in Travis because of this time out...

```
...........................................................................................................................................................................Timeout when running: python -c import bokeh.server; bokeh.server.run() --bokeh-port=5006 --backend=memory --splitjs
Starting Bokeh plot server on port 5006...
View http://127.0.0.1:5006/bokeh to see plots
F
======================================================================
FAIL: test_examples (examples.test_examples.TestExamples)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ContinuumIO/bokeh/examples/test_examples.py", line 10, in test_examples
    self.assertEqual(proc.wait(), 0)
AssertionError: 1 != 0
----------------------------------------------------------------------
Ran 179 tests in 7.735s
FAILED (failures=1)
The command "scripts/travis_test" exited with 1.
Done. Your build exited with 1.
```

--- a/examples/test
+++ b/examples/test
@@ -209,7 +209,7 @@ def start_bokeh_server():
         print("Failed to run: %s" % " ".join(cmd + argv))
         sys.exit(1)
     else:
-        def wait_until(func, timeout=1.0, interval=0.01):
+        def wait_until(func, timeout=5.0, interval=0.01):
             start = time.time()
 
             while True:

